__tests__/writer.test.js

```diff
@@ -0,0 +1,85 @@
+const fs = require('fs');
+const { write, writeJSON } = require('../writers/writer');
+const writePackageJSON = require('../writers/package-json-writer');
+const writeLinter = require('../writers/eslint-writer');
+const writeBabel = require('../writers/babel-writer');
+const writeWebPack = require('../writers/webpack-writer');
+const writeGitIgnore = require('../writers/git-ignore-writer');
+const writeTravis = require('../writers/travis-writer');
+const { writeIndexJS, writeIndexHTML } = require('../writers/src-index-writer');
+
+describe('writer.js', () => {
+  it('writes a text file', () => {
+    fs.writeFileSync = jest.fn();
+    let str = 'test string';
+    let path = '.';
+    write(path, str);
+    expect(fs.writeFileSync).toHaveBeenCalledWith(path, str);
```

Comment:
i dont think this i a thurough enough test. youre not testing any of the the functions here. you're only making sure they get called with the inputs you assigned, and nothing about what they actually do.